packages/flutter/lib/src/gestures/converter.dart

```diff
@@ -408,6 +409,8 @@ class PointerEventConverter {
               timeStamp: timeStamp,
               kind: kind,
               device: datum.device,
+              position: position,
+              delta: delta,
```

Comment:
If we already reported the delta on the hover event above, shouldn’t it be zero for the remove event then? My understanding is that it’s the delta of position since last event? (We should probably have a failing test for this situation...)

Reply:
Actually, I'm just going to remove the delta.  It's not clear what it's a delta from, since there isn't another event of the same type to diff it from, and nobody's expecting a delta anyhow. And not specifying it will set it to zero in any case.